xpdan/tests/pyFAI_calib.yml

```diff
@@ -11,7 +11,7 @@ centerY: !!python/object/apply:numpy.core.multiarray.scalar
   state: !!python/tuple [3, <, null, null, null, -1, -1, 0]
 - !!binary |
   ru+EJ0EKkEA=
-detector: Perkin detector
+detector: Perkin
```

Comment:
is it because pyFAI version we are testing one?

this yml was generated based on pyFAI output directly, so I would assume you make this change based on the pyFAI version we are testing on?